src/psyclone/tests/psyir/transformations/intrinsics/min2code_trans_test.py

```diff
@@ -80,17 +80,13 @@ def example_psyir_binary(create_expression):
 
     '''
     symbol_table = SymbolTable()
-    name1 = symbol_table.new_symbol_name("arg")
-    arg1 = DataSymbol(name1, REAL_TYPE, interface=ArgumentInterface(
-        ArgumentInterface.Access.READWRITE))
-    symbol_table.add(arg1)
-    name2 = symbol_table.new_symbol_name("arg")
-    arg2 = DataSymbol(name2, REAL_TYPE, interface=ArgumentInterface(
-        ArgumentInterface.Access.READWRITE))
-    symbol_table.add(arg2)
-    name3 = symbol_table.new_symbol_name()
-    arg3 = DataSymbol(name3, REAL_TYPE)
-    symbol_table.add(arg3)
+    arg1 = symbol_table.new_symbol(
```

Comment:
Please could you update the copyright date.

Reply:
Done